frame/balances/src/tests.rs

```diff
@@ -1167,6 +1183,20 @@ macro_rules! decl_tests {
 			});
 		}
 
+		#[test]
+		fn reserved_named_to_yourself_should_work() {
```

Comment:
Yeah, it's just a test that similar mechanic of moving reserved funds works well in `_Named` implementation.

As I can see, you follow flow of writing as short as it could be tests without any help-checks?

So some questions for best approach here:
- Should I remove `_named` test, if it works well on master, or just truncate it?
- (I don't think that's canonical and more suitable here, but...) Should I merge these tests into the one?

Reply:
Its mostly fine like this. I just mean that this test could be as long as the unnamed one.  
more checks = better